test/QT/up-to-date.py

```diff
@@ -134,7 +134,7 @@ def exists(env):
 
 expect = my_obj.replace( '/', os.sep )
 test.up_to_date(options = '--debug=explain',
-                arguments = (expect),
+                arguments =expect,
```

Comment:
is arguments expecting an array or tuple?

Reply:
see above.